core/src/ldml/C7043_ldml.md

```diff
@@ -324,10 +324,115 @@ For each key:
 - `vkey`: Is the standard vkey, 0-255
 - `target`: Is the target (resolved) vkey, 0-255.
 
+### C7043.2.x `lyrs`—Layers list
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+| 0 |  32  | ident      | `lyrs`                                   |
+| 4 |  32  | size       | int: Length of section                   |
+| 8 |  32  | count      | int: Number of layer lists               |
+|12 |  32  | layerstart | int: offset to first layer entry         |
+|16 |  32  | layercount | int: number of layer entries             |
+|20 |  96  | reserved   | padding                                  |
+
+Note layerstart must point after the end of the layer lists
+
+For each 'layers' - the layer lists
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | flags      | int: per-layers options                  |
+|24+|  32  | hardware   | str: layout (`us`,`iso`,`jis`,`abnt2`)   |
+|20+|  32  | target     | int: target vkey ID (0…255)              |
+|28+|  32  | layer      | int: index to first layer element        |
+|32+|  32  | layerCount | int: number of `layr` elements in list   |
+|36+|  96  | reserved   | short`[3]`: padding                      |
+
+- `flags`: a 32-bit bitfield defined as below:
+
+  | Bit position | Meaning  |  Description         |
+  |--------------|----------|----------------------|
+  |       0      | form     | 0: hardware          |
+  |       0      | form     | 1: touch             |
+
+For each layer element
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | id         | str: layer id such as `base` or `shift`  |
+|24+|  32  | modifier   | str: modifier string                     |
+|28+|  32  | row        | int: `rows` identifier of first row      |
+|32+|  32  | rowCount   | int: number of `rows` elements           |
```

Comment:
This implies a certain ordering in the `rows` section which I think is a little fragile. Either we pull `rows` into this `lyrs` section (my pref), or we rethink how we organise it.

Reply:
makes some sense.